qualtran/bloqs/gf_arithmetic/gf2_addition_test.py

```diff
@@ -0,0 +1,46 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from galois import GF
+
+from qualtran.bloqs.gf_arithmetic.gf2_addition import (
+    _gf2_addition_symbolic,
+    _gf16_addition,
+    GF2Addition,
+)
+from qualtran.testing import assert_consistent_classical_action
+
+
+def test_gf16_multiplication(bloq_autotester):
```

Comment:
multiplication?

Reply:
Reworded to addition, a copy/paste error.